commands/core/config.drush.inc

```diff
@@ -109,10 +109,8 @@ function config_drush_command() {
         'example-value' => 'branchname',
       ),
       'destination' => 'An arbitrary directory that should receive the exported files. An alternative to label argument.',
-      'skip-modules' => 'A list of modules to ignore during export (e.g. to avoid listing dev-only modules in exported configuration).',
```

Comment:
That makes sense if the value was 0. How frequent is that though? If the value actually listed modules, it makes sense to error out and ideally discuss config-split. All in all, I'm OK with this PR as is.